README.md

```diff
@@ -1,6 +1,6 @@
 # @zeit/tracing-js 
 
-[![npm](https://badgen.net/npm/v/@zeit/tracing-js)](https://www.npmjs.com/package/@zeit/tracing-js) [![install size](https://badgen.net/packagephobia/install/@zeit/tracing-js)](https://packagephobia.now.sh/result?p=@zeit/tracing-js) [![codecov](https://badgen.net/codecov/c/github/zeit/tracing-js)](https://circleci.com/gh/zeit/workflows/tracing-js)
+[![npm](https://badgen.net/npm/v/@zeit/tracing-js)](https://www.npmjs.com/package/@zeit/tracing-js) [![install size](https://badgen.net/packagephobia/install/@zeit/tracing-js)](https://packagephobia.now.sh/result?p=@zeit/tracing-js) [![codecov](https://badgen.net/circleci/github/zeit/tracing-js)](https://circleci.com/gh/zeit/workflows/tracing-js)
```

Comment:
Would be good to change the alt-text from `codecov` to `circleci` too. Other than that LGTM

Reply:
Good point, done!

I still cant figure out why codecov doesn't work and their support channel is too busy to respond 🤷‍♂